volatility/framework/symbols/windows/extensions/__init__.py

```diff
@@ -402,6 +400,25 @@ def owning_process(self, kernel_layer: str = None) -> interfaces.objects.ObjectI
         """Return the EPROCESS that owns this thread."""
         return self.ThreadsProcess.dereference(kernel_layer)
 
+      def get_cross_thread_flags(self) -> str:
+        dictCrossThreadFlags = {'PS_CROSS_THREAD_FLAGS_TERMINATED': 0,
+                                'PS_CROSS_THREAD_FLAGS_DEADTHREAD': 1,
+                                'PS_CROSS_THREAD_FLAGS_HIDEFROMDBG': 2,
+                                'PS_CROSS_THREAD_FLAGS_IMPERSONATING': 3,
+                                'PS_CROSS_THREAD_FLAGS_SYSTEM': 4,
+                                'PS_CROSS_THREAD_FLAGS_HARD_ERRORS_DISABLED': 5,
+                                'PS_CROSS_THREAD_FLAGS_BREAK_ON_TERMINATION': 6,
+                                'PS_CROSS_THREAD_FLAGS_SKIP_CREATION_MSG': 7,
+                                'PS_CROSS_THREAD_FLAGS_SKIP_TERMINATION_MSG': 8}
+        
+        flags = self.CrossThreadFlags
+        stringCrossThreadFlags = ''
+        for flag in dictCrossThreadFlags:
+            if flags & 2**dictCrossThreadFlags[flag]:
```

Comment:
We'd typically do `(1 << dictCrossThreadFlags[flag])`, but it's definitely not a blocker